.github/workflows/test-php.yml

```
@@ -66,3 +66,28 @@ jobs:
+      - name: Checkout code
+        uses: actions/checkout@3d3c42e5aac5ba805825da76410c181273ba90b1 # v7
+
+      - name: Setup PHP with mbstring disabled
+        uses: shivammathur/setup-php@f3e473d116dcccaddc5834248c87452386958240 # v2
+        with:
+          php-version: ${{ matrix.php-versions }}
+          extensions: ':mbstring'
+          coverage: none
+
+      - name: Assert the mbstring extension is absent
+        run: php -r "exit(extension_loaded('mbstring') || function_exists('mb_strtolower') ? 1 : 0);"
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)